src/ExperimentalCycles.mo

```diff
@@ -1,51 +1,152 @@
-/// Managing cycles
+/// Managing cycles within actors on the Internet Computer (IC).
 ///
-/// Usage of the Internet Computer is measured, and paid for, in _cycles_.
+/// The usage of the Internet Computer is measured, and paid for, in _cycles_.
 /// This library provides imperative operations for observing cycles, transferring cycles and
 /// observing refunds of cycles.
 ///
 /// **WARNING:** This low-level API is **experimental** and likely to change or even disappear.
 /// Dedicated syntactic support for manipulating cycles may be added to the language in future, obsoleting this library.
 ///
-/// **NOTE:** Since cycles measure computational resources, the value of
-/// `balance()` can change from one call to the next.
-
+/// **NOTE:** Since cycles measure computational resources, the value of  `balance()` can change from one call to the next.
+///
+/// Example for use on IC:
+/// ```motoko
+/// import Cycles "mo:base/ExperimentalCycles";
+/// import Debug "mo:base/Debug";
+///
+/// actor {
+///     public func main(): async() {
+///         Debug.print("Main balance: " # debug_show(Cycles.balance()));
+///         Cycles.add(15_000_000);
+///         await operation(); // accepts 10_000_000 cycles
+///         Debug.print("Main refunded: " # debug_show(Cycles.refunded())); // 5_000_000
+///         Debug.print("Main balance: " # debug_show(Cycles.balance())); // decreased by around 10_000_000
+///     };
+///
+///     func operation(): async() {
+///         Debug.print("Operation balance: " # debug_show(Cycles.balance()));
+///         Debug.print("Operation available: " # debug_show(Cycles.available()));
+///         let obtained = Cycles.accept(10_000_000);
+///         Debug.print("Operation obtained: " # debug_show(obtained)); // => 10_000_000
+///         Debug.print("Operation balance: " # debug_show(Cycles.balance())); // increased by 10_000_000
+///         Debug.print("Operation available: " # debug_show(Cycles.available())); // decreased by 10_000_000
+///     }
```

Comment:
```suggestion
///   public func main() : async() {
///     Debug.print("Main balance: " # debug_show(Cycles.balance()));
///     Cycles.add(15_000_000);
///     await operation(); // accepts 10_000_000 cycles
///     Debug.print("Main refunded: " # debug_show(Cycles.refunded())); // 5_000_000
///     Debug.print("Main balance: " # debug_show(Cycles.balance())); // decreased by around 10_000_000
///   };
///
///   func operation() : async() {
///     Debug.print("Operation balance: " # debug_show(Cycles.balance()));
///     Debug.print("Operation available: " # debug_show(Cycles.available()));
///     let obtained = Cycles.accept(10_000_000);
///     Debug.print("Operation obtained: " # debug_show(obtained)); // => 10_000_000
///     Debug.print("Operation balance: " # debug_show(Cycles.balance())); // increased by 10_000_000
///     Debug.print("Operation available: " # debug_show(Cycles.available())); // decreased by 10_000_000
/// }
```

Whitespace changes to use 2-space indentation and have whitespace before and after `:` in type annotations.
From https://internetcomputer.org/docs/current/developer-docs/build/cdks/motoko-dfinity/style

Reply:
I think similar changes should be applied to the other functions

Reply:
Thanks a lot. I will check the other functions.